packages/apps/job-launcher/server/src/common/enums/webhook.ts

```diff
@@ -1,7 +1,7 @@
 export enum EventType {
   ESCROW_CREATED = 'escrow_created',
   ESCROW_CANCELED = 'escrow_canceled',
-  TASK_CREATION_FAILED = 'task_creation_failed',
```

Comment:
The event type for this case is not submissions rejected, it is escrow_failed

Reply:
Changed